bugshaker/src/main/java/com/github/stkent/bugshaker/MainActivity.java

```diff
@@ -180,7 +240,7 @@ public void onClick(DialogInterface dialog, int which) {
 						, SharedPreferencesUtil.getEmailSubjectLine(getBaseContext()));
 					savedToast.show();
 
-					LogcatUtil.saveLogcatToFile(getApplicationContext());
+//					LogcatUtil.saveLogcatToFile(getApplicationContext());
```

Comment:
Can we remove this comment? Also, does this not affect us saving the logcat file?